perllib/FixMyStreet/DB/ResultSet/Problem.pm

```diff
@@ -172,6 +172,10 @@ sub around_map {
             latitude => { '>=', $p{min_lat}, '<', $p{max_lat} },
             longitude => { '>=', $p{min_lon}, '<', $p{max_lon} },
     };
+
+    my $interval = $p{report_age};
```

Comment:
Not sure `$interval` var needed here, isn't one in Nearby.pm (or if you do want to use it both times in the line underneath rather than only once :) )